client/src/pages/workspace/cardPanel/addCard/AddCard.tsx

```diff
@@ -0,0 +1,63 @@
+import React, {useEffect, useState} from 'react';
+import Button from "@mui/material/Button";
+import {Card, CardContent, TextField, Typography} from "@mui/material";
+import {AddCardProp} from './props';
+import {ICardModification} from "../../../../models/Card";
+import "./styles.css";
+
+const AddCard: React.FC<AddCardProp> = ({boardId, onCreate}: AddCardProp) => {
+    const [card, setCard] = useState<ICardModification>({name: '', description: ''})
+    const [valid, setValid] = useState<boolean>(false)
+
+    useEffect(() => {
+        isValid(card);
+    }, [card, boardId]);
+
+    const isValid = (card: ICardModification) => {
```

Comment:
Such way you need to name a variable. Methods are actions.
```suggestion
    const validateCard = (card: ICardModification) => {
```